src/content/developers/docs/smart-contracts/testing/index.md

```diff
@@ -1,37 +1,204 @@
+- [Documentation](https://remix-ide.readthedocs.io/en/latest/unittesting.html) 
+- [GitHub](https://github.com/ethereum/remix-project/tree/master/libs/remix-tests)  
+
+**OpenZeppelin Test Helpers** - _Assertion library for Ethereum smart contract testing. Make sure your contracts behave as expected!_
+- [GitHub](https://github.com/OpenZeppelin/openzeppelin-test-helpers) 
+- [Documentation](https://docs.openzeppelin.com/test-helpers) 
+
+**Truffle smart contract test framework** - _Automated testing framework to make testing your contracts a breeze._
+- [Documentation](https://trufflesuite.com/docs/truffle/testing/testing-your-contracts/) 
+- [Website](https://trufflesuite.com/) 
+
+**Brownie unit testing framework** - _Brownie utilizes Pytest, a feature-rich test framework that lets you write small tests with minimal code, scales well for large projects, and is highly extendable._
+- [Documentation](https://eth-brownie.readthedocs.io/en/v1.0.0_a/tests.html) 
+- [GitHub](https://github.com/eth-brownie/brownie) 
+
+### Static analysis tools {#static-analysis-tools}
+ 
+**Mythril** - _EVM bytecode assessment tool for detecting contract vulnerabilities using taint analysis, concolic analysis, and control flow checking._
+
+- [GitHub](https://github.com/ConsenSys/mythril-classic) 
+- [Documentation](https://mythril-classic.readthedocs.io/en/master/about.html) 
+
+**Manticore** - _A static analysis tool for analyzing EVM bytecode based on symbolic execution._
+- [GitHub](https://github.com/trailofbits/manticore)  
+- [Documentation](https://github.com/trailofbits/manticore/wiki) 
 
-- [Whiteblock.io](https://whiteblock.io)
-- [Documentation](https://docs.whiteblock.io)
-- [GitHub](https://github.com/whiteblock/genesis)
+### Dynamic analysis tools {#dynamic-analysis-tools}
 
-**OpenZeppelin Test Environment -** **_Blazing fast smart contract testing. One-line setup for an awesome testing experience._**
+**Echidna** - _A fast contract fuzzer for detecting vulnerabilities in smart contracts through property-based testing._
+- [GitHub](https://github.com/crytic/echidna/) 
 
-- [GitHub](https://github.com/OpenZeppelin/openzeppelin-test-environment)
-- [Documentation](https://docs.openzeppelin.com/test-environment/)
+**Harvey** - _Automated fuzzing tool useful for detecting property violations in smart contract code._
+- [Website](https://consensys.net/diligence/fuzzing/)
 
-**OpenZeppelin Test Helpers -** **_Assertion library for Ethereum smart contract testing. Make sure your contracts behave as expected!_**
+### Smart contract auditing services {#smart-contract-auditing-services}
 
-- [GitHub](https://github.com/OpenZeppelin/openzeppelin-test-helpers)
-- [Documentation](https://docs.openzeppelin.com/test-helpers)
+**ConsenSys Diligence** - _Helps projects across the blockchain ecosystem to ensure their protocols are ready for launch and built to protect users._
+- [Website](https://consensys.net/diligence/) 
+
```

Comment:
Add trail of bits (they make https://github.com/trailofbits/eth-security-toolbox)

Reply:
> Add trail of bits (they make https://github.com/trailofbits/eth-security-toolbox)

Trail of Bits is listed in this section already.